src/constants/transactionTypes.js

```diff
@@ -0,0 +1,7 @@
+export default {
```

Comment:
we’re creating `src/constants/transactionTypes.js` when there are some other places to use this but you only uses this in one file.So, we don't need this constant.

Reply:
I use it in three files:
```
src/components/transactions/amount.js
src/store/middlewares/delegateRegistration.js
src/store/middlewares/delegateRegistration.test.js
```